tools-local/DefaultGenApiDocIds.txt

```diff
@@ -0,0 +1,6 @@
+// These attributes should be excluded from reference assemblies.
+
```

Comment:
See https://github.com/dotnet/standard/blob/master/platforms/attributeExcludeList.txt for the complete set that I use for standard.